runtime/compiler/runtime/RuntimeAssumptions.hpp

```diff
@@ -147,6 +160,9 @@ class TR_PersistentClassInfo : public TR_Link0<TR_PersistentClassInfo>
    int32_t getNameLength()                       { return _nameLength; }
    virtual void setNameLength(int32_t length)            { _nameLength = length; }
 
+   void setCCVResult(CCVResult result) { _ccvResult = static_cast<uint8_t>(result); }
+   const CCVResult getCCVResult() { return static_cast<CCVResult>(_ccvResult); }
```

Comment:
`const` should stay after the name of the function to indicate that the function does not change the object it operates on.